### ISSUE TITLE
New Lore Notifications

### ISSUE THREAD

**Issue by rozzingit** [Arx-Game/arxcode#109]
**Is your feature request related to a problem? Please describe.**
The `lore` command is totally a better way to handle these questions than the Theme Questions bboard. BUT now there is no notification about new lore files unless Apos manually posts something to let people know.

**Describe the solution you'd like**
Maybe a Lore or Lore Updates bboard? That just has an automated posting when staff publishes a new lore file to be like `A new lore file on [TOPIC] has been posted. It can be read at 'lore [TOPIC]' or at [URL].` (once the lore files exist on the website for that URL).

**Describe alternatives you've considered**
You could also do this with maybe @informs if you do some sort of way to opt in or out of them? I figure bboards might be simpler because the process of opting in or out there is already baked-in.

**Comment by Sritterbush** [comment]
I love everything about this suggestion.

**Comment by TehomCD** [comment]
Yeah, maybe a board would be easiest. I kinda feel like this combined with roster changes begin to approach a problem with too many boards that all serve the purpose of 'notify me when staff does X thing', and we'll need a more general solution to that eventually - like some sort of subscription-based notification system of changes since your last login via whatever or being sent a message if you're online.

**Comment by rozzingit** [comment]
Yeah, I can definitely understand the desire to move away from using bboards for this purpose. I think they're a pretty reasonable stopgap in the meantime, though, even if you want to make a new system down the road. And people can subscribe and unsubscribe at their leisure. And I do think the best part of new lore entries is people knowing to read them! jk the best part is me reading them

**Comment by rozzingit** [comment]
I feel like right now staff sometimes thinks to make a manual bbpost about new lore files, but most of them are kind of going into the ether, which is a real shame, as several of the ones I've heard about are actually directly relevant to ongoing IC debate (like a few thralldom ones) that would be good points of OOC clarification. If there's a way to set it up to bbpost on a new board without too much trouble, I think it'd be really helpful.
